zip-0226.rst

```diff
@@ -1,8 +1,313 @@
 ::
 
   ZIP: 226
-  Title: Zcash Shielded Assets - Transfers and Burns
-  Owners: Daniel Bennaroch
-  Status: Reserved
+  Title: Transfer and Burn of Zcash Shielded Assets
+  Owners: Daniel Benarroch <daniel@qed-it.com>
+          Pablo Kogan <pablo@qed-it.com>
+          Aurelien Nicolas <aurel@qed-it.com>
+  Credits: Daira Hopwood
+           Jack Grigg
+           Vivek Arte
+  Status: Draft
   Category: Consensus
+  Created: 2022-05-01
+  License: MIT
   Discussions-To: <https://github.com/zcash/zips/issues/618>
+
+
+Terminology
+===========
+
+The key word "MUST" in this document is to be interpreted as described in RFC 2119 [#RFC2119]_.
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_.
+
+The term "Orchard" in this document is to be interpreted as described in ZIP 224 [#zip-0224]_.
+
+We define the following additional terms:
+
+- Asset: A type of note that can be transferred on the Zcash block chain, identified by the :math:`\mathsf{AssetId}` parameter.
+
+  - ZEC is the default (and currently the only defined) Asset for the Zcash mainnet.
+  - TAZ is the default (and currently the only defined) Asset for the Zcash testnet.
+  - We use the term "Custom Asset" to refer to any Asset other than ZEC and TAZ.
+
+- Native Asset: a Custom Asset with issuance defined on the Zcash block chain.
+- Wrapped Asset: a Custom Asset with native issuance defined outside the Zcash block chain.
+- Split Input: an Action input identifying a Custom Asset note, used to ensure that the output note of that Action is of a validly issued :math:`\mathsf{AssetId}` when there is no corresponding real input note to spend. The Action does not spend the note identified by the Split Input.
+- Split Action: an Action that contains a Split Input.
+
+Abstract
+========
+
+ZIP 226 and ZIP 227 propose in conjunction the Zcash Shielded Assets (ZSA) protocol &mdash; a set
+of protocol features that enable the creation, transfer, and burn of Custom Assets on the Zcash chain.
+
+Creation of such Assets is defined in ZIP 227 [#zip-0227]_. Transfer and burn of such Assets is defined
+in ZIP 226 [#zip-0226]_. The ZSA protocol is proposed to be instantiated by a modification to the
+Orchard protocol, as specified in these ZIPs.
+
+Motivation
+==========
+
+None of the currently deployed Zcash transfer protocols support Custom Assets. Enabling
+multi-asset support on the Zcash chain will open the door for a host of applications, and
+enhance the ecosystem with application developers and Asset custody institutions for
+issuance and bridging purposes.
+
+Overview
+========
+In order to be able to represent different Asset identifiers in Orchard, we need to define a new data field that uniquely represents the identifier of the Asset in question, which we call :math:`\mathsf{AssetId}`.
```

Comment:
Anywhere where we haven't specified something else, please specify :math:`\mathsf{AssetId^{Orchard}}` vs the general, shielded protocol-agnostic :math:`\mathsf{AssetId}`

Reply:
Has been subsequently replaced by `AssetBase`, but have used the Orchard specific terminology where necessary.